src/topojsonm/topojsonm.ml

```diff
@@ -171,16 +171,17 @@ let map_object f src dst =
   in
   let rec go () =
     match Jsonm.decode decoder with
-    | `Lexeme (`Name "object" as t) ->
+    | `Lexeme (`Name "objects" as t) ->
         (match Jsonm.decode decoder with
-        | `Lexeme (`Name "example" as ex) -> (
+        | `Os -> (
```

Comment:
Remember we're also building the output as we go, so you will need to encode these values. 

Unfortunately the API was built with polymorphic variants so there's no error here but `Jsonm.decode` cannot return just `` `Os`` it has to be `` `Lexeme `Os``